pkg/exec/error.go

```diff
@@ -128,3 +128,19 @@ func IsTimeOutError(err error) bool {
 
 	return false
 }
+
+func CheckEOFError(_ interface{}, err error) ErrorType {
```

Comment:
We have here `pkg/exec/error.go:105` an EOF check already..  
Why is it not working anymore?